modules/ref_fb_module/src/fft_fb_impl.cpp

```diff
@@ -186,10 +186,10 @@ void FFTFbImpl::configure()
     }
     catch (const std::exception& e)
     {
-        setComponentErrorStateStatusWithMessage(ComponentErrorState::Warning, "Failed to set descriptor for signal");
-        LOG_W("FFT: Failed to set descriptor for signal: {}", e.what())
+        setComponentStatusWithMessage(ComponentStatus::Warning, fmt::format("FFT: Failed to set descriptor for signal: {}", e.what()));
         outputSignal.setDescriptor(nullptr);
     }
+    setComponentStatus(ComponentStatus::Ok);
```

Comment:
Considering we always catch exceptions, we will always override the component status with "Ok" at the end here.